### ISSUE TITLE
For numbers summed to 100%, there can be no such thing that only one member doesn't follow Benford's rule.

### ISSUE THREAD

**Issue by szabolcsvelkei** [cjph8914/2020_benfords#13]
Keep calm and learn math.

ps.: don't use lead digits in these situation. Misleading.

**Comment by jimfcarroll** [comment]
It appears he ran these on the vote counts, not the percentages. It would be absurd to run these on the percentages anyway since the orders of magnitude don't vary enough.

**Comment by dshield55** [comment]
This may not be an issue and may actually already be showing up as true.

Look at Chicago. Biden has a spike for 3,4,5s and Trumps numbers after 2 are completely flat. Spikes and even distributions both hint at fraud, wouldn't that indicate that both candidates have been tampered with? Possibly Trump votes moved to Biden?

In one of the swing states where Biden overcomes Trump, there are 70,000 voters who voted for a President, but no Senator. This patter repeats in several swing states, but not in non-swing states. If at a precinct level, votes were being added to Biden but not transferred from Trump, wouldn't that allow only one member to violate Benford's Law since we're not dealing with a 100% sum problem anymore?

**Comment by szabolcsvelkei** [comment]
Voting numbers, percentages, doesn't matter.  These numbers are not independent:
Sum of votes: N=n1+n2+n3+n4..., so if you cheat on one column (like n1), you must cheat at least one other too. (n2 or something)
This is a "naive" common sense falsification, which anyone can understand without being mathematically skilled.

Of course there are a lot of other mathematical incorrent things in this situation. (Using Benford in voting situation like this is one of them. :) )

**Comment by jimfcarroll** [comment]
Yes. I was just editing my comment when your last response came up. What @dshield55 says in his last paragraph is what I meant. They ARE independent if ballot-box stuffing is what took place. It's NOT like they took votes from Trump and moved them to Biden. The claim is they simply added (i.e effectively made up) Biden counts.

In your example you have N fixed and so any anomaly in one distribution (say n1) would affect another. If I add a distribution that violates Benford's law (say n1), to distributions (of the same orders of magnitude) that don't (n2, n3), the additional anomaly doesn't appear in the other distributions, it appears in the totals (N in this case).

What's interesting about this, if, hypothetically what I describe above is what took place, you would expect the totals to be unknown prior to the fraud taking place. Interestingly enough Rich Baris from BigData Polling was complaining about this in real time.

**Comment by dshield55** [comment]
> Interestingly enough Rich Baris from BigData Polling was complaining about this in real time.

Can you add more on that?

**Comment by jimfcarroll** [comment]
Rich was complaining the totals kept changing as the night went on. He kept mentioning that this was the first time he'd ever seen it. They normally report the totals BEFORE the counting. That's how the "Percent complete" is known. In this case the swing state provided very wide ranges up front. Then, as the night went on, they kept upping even the high end of these ranges. Dramatically in some cases. Rich was commenting on this throughout the night 

I've been following him for a couple months since he nailed 2016. On the night of the election he was a repeat guest on two livestreams I kept bouncing between trying to get his take (I didn't care about anything else either stream had to say).

**Comment by jimfcarroll** [comment]
If you're interesred he recaps his experience in the first 15 minutes of this Nov 5th livestream: https://www.youtube.com/watch?v=dEe463Y5lAw

**Comment by jimfcarroll** [comment]
FWIW, this seems to be the best refutation: https://github.com/cjph8914/2020_benfords/issues/9 We'd need to see if the analysis persists across more than Allegheny

**Comment by mlewis1973** [comment]
> It appears he ran these on the vote counts, not the percentages. It would be absurd to run these on the percentages anyway since the orders of magnitude don't vary enough.

even the vote counts don't have enough orders of magnitude

**Comment by szabolcsvelkei** [comment]
@jimfcarroll,  changing the total number of votes would be the most lame fraud and easiest to catch. No fool says that 1000 votes are in the box when there are only 800.

**Comment by jimfcarroll** [comment]
@szabolcsvelkei  I think you misunderstood me. They didn't just "change the total."  In typical ballot-stuffing schemes it's important to have flexibility in the total numbers because if you underestimate what you need, you can increase the totals on the fly as you stuff more prepared ballots. In a canvasing operation the ballots will be there.

It helps to pause the counting to asses what's needed. Stopping or "slow walking" the count was done in several well known suspect elections including by Brenda Snipes in Broward county FL and during Scott Walker's election in WI. Brenda Snipes was removed by the then governor Rick Scott in 2018 after she was slow walking the counting and refused to declare what the ballot totals were. (see: https://www.sun-sentinel.com/news/politics/fl-ne-broward-election-audit-2018-brenda-snipes-20200526-rswvxtj2ljasjmfi7w75moo52e-story.html where a 2018 audit found "unnecessary delays" and https://www.nationalreview.com/news/woman-overseeing-broward-vote-count-illegally-destroyed-ballots-in-previous-race/ where she refused to provide totals).

**Comment by JivanRoquet** [comment]
@szabolcsvelkei changing the total number of votes didn't seem to pose them a problem. Furthermore, I don't see why one wouldn't apply Benford's rules in this situation as the OP did. It is perfectly valid the way he did it and proof is that the law is effectively respected for most of the curves, while not for a few others. You're trying to teach the OP out of it by using a somewhat arrogant and condescending tone, but it doesn't quite add up.

**Comment by jimfcarroll** [comment]
@JivanRoquet I thought this also, originally. However, I think @MechanicalTim provides a reasonable explanation at issue https://github.com/cjph8914/2020_benfords/issues/9

**Comment by szabolcsvelkei** [comment]
@JivanRoquet, sorry for my somehow arrogant style, it is my fault.
My outrage stems from the fact that the author uses math as a means of reasoning, i.e. he refers to authority.  He/she has no name, just a freshly add 1 days old nick in github with only one repo. These are the characteristics of a troll. In addition, he has preconceptions and look for data for them, not the other way. He WANTS to prove that Biden cheated and he is not independent.

But the fact is that wherever Trump wins by a similar proportion, Trump's votes look like they're breaking the Benford rule. So two final options remained:
1. everyone totally cheated on the election
2. the approach by which the author tries to prove cheating is perfectly wrong

Which is more likely?

**Comment by cristi-neagu** [comment]
I don't see what this has to do with the code itself. Whether or not there was fraud or whether or not these notebooks show fraud has nothing to do with the code since the claim was never made that they show fraud. This is just a first digit analysis of the published data. It makes no interpretation.

**Comment by ghost** [comment]
In science, charts are always properly discussed. Just presenting charts without any context will easily be misleading.

**Comment by JivanRoquet** [comment]
@szabolcsvelkei makes sense. I'm personally 99% certain that there was indeed massive fraud from the Dems side, but I agree with you on all the points you made. Notably I'm absolutely not convinced that Benford's law can be used the way it is used here, let alone be a sufficient argument to jump to any kind of conclusion.

Furthermore I would actually point that if fraud there was, then using spurious "data analysis" is a very good way to discredit any other serious and valid arguments.
